ext/redhat/logrotate

```diff
@@ -1,9 +1,21 @@
-/var/log/puppetlabs/puppet.log {
+# Generic logrotate config for Red Hat systems. Works with both official
+# Fedora and RHEL packages as well as PuppetLabs distributions.
+
+/var/log/puppetlabs/masterhttp.log /var/log/puppet/masterhttp.log {
+  compress
   missingok
   notifempty
-  create 0644 puppet puppet
+  nocreate
+}
+
+/var/log/puppetlabs/puppetd.log /var/log/puppet/puppetd.log {
+  compress
+  missingok
+  notifempty
+  nocreate
   sharedscripts
   postrotate
-    [ -e /etc/init.d/puppet ] && /etc/init.d/puppet reload > /dev/null 2>&1 || true
+   ([ -e /etc/init.d/puppet ] && /etc/init.d/puppet reload > /dev/null 2>&1) ||
```

Comment:
nitpick. the test here should be `-x` as we are executing the init script directly.